x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/action/TransportValidateDetectorAction.java

```diff
@@ -18,8 +18,7 @@
 
     @Inject
     public TransportValidateDetectorAction(TransportService transportService, ActionFilters actionFilters) {
-        super(ValidateDetectorAction.NAME, transportService, ValidateDetectorAction.Request::new, actionFilters
-        );
+        super(ValidateDetectorAction.NAME, transportService, ValidateDetectorAction.Request::new, actionFilters);
```

Comment:
Should this super ctor be swapped as well?

Reply:
Nope, surprisingly this class was not converted to writeable yet